syntax = "proto3";

package com.project.groupService;

message GroupResponse {
  int64 id = 1;
  string title = 2;
}

message GroupRequest {
  int64 groupId = 1;
}

message GroupTitleRequest {
  string title = 1;
}

service GroupService {
  rpc getGroupByGroupId(GroupRequest) returns (GroupResponse);
  rpc getGroupByTitle(GroupTitleRequest) returns (GroupResponse);
}